### PR TITLE
Allow U2F second factors to be vetted

### PR DESCRIPTION
**Pull request by rjkip**
- [x] Merge #96.
 - [x] Merge into `develop` via CLI. (PR is created against `feature/allow-enabling-of-sfs` for the diff's sake)

### CODE REVIEW

**Comment by DRvanR** [comment]
:+1:

**Comment by rjkip** [comment]
:8ball: @DRvanR

File: src/Surfnet/StepupRa/RaBundle/Command/CreateU2fSignRequestCommand.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\StepupRa\RaBundle\Command;
+
+final class CreateU2fSignRequestCommand
+{
+    // FIXME
```
Comment (DRvanR) [inline src/Surfnet/StepupRa/RaBundle/Command/CreateU2fSignRequestCommand.php]: U WOT M8? :wink:

**Comment by DRvanR** [comment]
:+1:

**Comment by rjkip** [comment]
Merged.

**Pull request closed without merging**